src/main/java/org/folio/invoices/utils/ErrorCodes.java

```diff
@@ -21,7 +21,7 @@ public enum ErrorCodes {
   INCORRECT_FUND_DISTRIBUTION_TOTAL("incorrectFundDistributionTotal","Fund distribution total must add to 100% or totalPrice"),
   CANNOT_MIX_TYPES_FOR_ZERO_PRICE("cannotMixTypesForZeroPrice", "Fund distribution types cannot be mixed for a price of 0"),
   ADJUSTMENT_FUND_DISTRIBUTIONS_SUMMARY_MISMATCH("adjustmentFundDistributionsSummaryMismatch", "Fund distributions summary should be 100 % or equal to subtotal for every non-prorated adjustment"),
-  VOUCHER_UPDATE_FAILURE("voucherUpdateFailure", "Voucher record cannot be updated"),
+  VOUCHER_UPDATE_FAILURE("voucherUpdateFailure", "Voucher record cannot be updated: %s"),
```

Comment:
CC: @damien-git

Reply:
In general it is good to use parameter. The problem with them is that there is only one level of parameters, so if you want to report an error that happened in another module that is using parameters and you want to put that error in a parameter, you have to also add the parameters of that error to the new parameter value. I thought it was simpler to create long messages including the cause because that works with no limit in depth. But that is only for errors happening in other modules that are caught in this module.

Reply:
Thanks to clarification guys, I moved String message to parameters as Serhii suggested